test/test_http_instrumentation.py

```diff
@@ -340,6 +402,13 @@
 BASE_PATH = os.path.dirname(os.path.realpath(__file__))
 
 
+def fix_about_page_url(url):
+    if url == "about:blank":
```

Comment:
This looks like an issue in the extension. The `about:blank` URL should not load any sub resources.

I'm guessing this is an instance of https://github.com/mozilla/OpenWPM/issues/245. I'm curious, was this reliably incorrect for some requests, or did it seem to be a race condition?

Reply:
Mind to add a comment pointing to that issue?

Reply:
My very early diagnosis for this is that XUL browsers start off with about:blank (there was some perf work going on to prevent unnecessary about:blank loads in new windows for example). I'll add a comment pointing to the issue.